presto-docs/src/main/sphinx/release/release-0.285.rst

```diff
@@ -0,0 +1,75 @@
+=============
+Release 0.285
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix `min_by(x, y, n) / max_by(x, y, n)` functions to ensure `n` is unique.
+* Fix LIKE pattern to handle multi-byte characters. :pr:`21578`
+* Improve latency of HBO. :pr:`21297`
+* Improve HBO to track partial aggregation statistics. :pr:`21160`
+* Improve PushPartialAggregationThroughExchange to use partial aggregation statistics when available. This extends to multi-key aggregations.
+* Improve union node by removing the empty inputs.
+* Improve HBO latency by skipping retry if it times out. :pr:`21308`
+* Improve join-reordering to work with non-simple equi-join predicates.
```

Comment:
```suggestion
* Improve cost based optimizer join reordering to work with ....
```

Any suggestions on how to make "non-simple equijoin predicates" a little bit more understandable?

Reply:
Probably a link to the PR will help.

Reply:
CC: @aaneja for input.

Reply:
How about - `Improve join reordering for complex equi-join clauses` PR : https://github.com/prestodb/presto/pull/21153

Reply:
`complex` is slightly better than `non-simple` but I believe a Presto user is better off looking at the PR for the actual improvement. I don't think there is an easy wording here.